<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome to Sisyphus Setup</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/sisyphus.png</normaloff>../icon/sisyphus.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>482</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="wizard">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="text">
       <string>Welcome to Sisyphus!

This is your first time running the application. 
Please configure your settings to continue.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="abortButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Abort</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icon/exit.png</normaloff>../icon/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="configureButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Configure</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icon/settings.png</normaloff>../icon/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
